test_project/urls.py

```diff
@@ -2,10 +2,10 @@
 from django.contrib import admin
 
 from django_graph_api.views import GraphQLView
-from test_app.schema import schema
+from test_app.schema import QueryRoot
 
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^graphql$', GraphQLView.as_view(schema=schema)),
+    url(r'^graphql$', GraphQLView.as_view(query_root_class=QueryRoot)),
```

Comment:
It's not a big deal, but I think it still makes sense to pass in the schema object, since it will eventually have mutations and directives as well as queries.

Reply:
My plan for that would be that additional items would also be passed to GraphQLView alongside query_root_class.